etc/update-todo.sh

```diff
@@ -1,18 +1,23 @@
 #!/bin/bash
 
-rg -h 1>/dev/null || (echo "ripgrep not found" && false)
+# exit on error (-e), unset var (-u), and failed command in pipe line (-o pipefail)
+set -euo pipefail
 
-IFS=$'\n'
+command -v rg 1>/dev/null || (echo 2>"error: ripgrep not found" && false)
 
-mv TODO.md TODO.bak || true
+mv TODO.{md,bak} || true
 touch TODO.md
 
-for FILE in $(rg '@TODO' --ignore-file='update-todo.sh' --files-with-matches); do
+IFS=$'\n'
+for FILE in $(rg '@TODO|FIXME' --glob='!update-todo.sh' --files-with-matches); do
```

Comment:
Can we remove the `@` here? I use `TODO:` because VS code supports only that specific.

Reply:
Yeah that's an easy fix.